README.md

```diff
@@ -1,43 +1,96 @@
-# Iroha 2
+# Overview
 
 A very simple and performant blockchain.
 
-## Description
+## About
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 ![Rust](https://github.com/hyperledger/iroha/workflows/Rust/badge.svg?branch=iroha2-dev)
 [![codecov](https://codecov.io/gh/hyperledger/iroha/branch/iroha2-dev/graph/badge.svg)](https://codecov.io/gh/hyperledger/iroha)
 
 
-Iroha is a straightforward distributed ledger technology (DLT), inspired by Japanese Kaizen principle — eliminate excessiveness (muri). Iroha has essential functionality for your asset, information and identity management needs, at the same time being an efficient and trustworthy crash fault-tolerant tool for your enterprise needs.
+Iroha is a straightforward distributed ledger technology (DLT), inspired by the Japanese Kaizen principle — eliminate excesses (muri). Iroha has essential functionality for your asset, information and identity management needs, while also being an efficient and trustworthy crash and fault-tolerant tool for your enterprise needs.
 
-### Features
+## Features
 
 Iroha has the following features:
 
-* Creation and management of custom fungible assets, such as currencies, kilos of gold, etc.
-* Management of user accounts
-* Taxonomy of accounts based on domains in the system
+* Creation and management of custom fungible assets, such as currencies, gold, etc.
+* User account management including domain-based taxonomy
 * The system of rights and verification of user permissions for the execution of transactions and queries in the system
+* Byzantine fault-tolerance with up to 34% fault rate
 
-## Usage
+# System Requirements
 
-### Requirements
+## Building
+### Minimum
+
+A modern multi-core CPU with at least 2GB of RAM per core as well as approximately 20 GB of free storage space.
+
+### Recommended
+
+Ryzen 5 1600 or above, 16GB of RAM or more. 40GB of SSD storage.
+
+## Running
+
+The actual minimum requirements depend on the load that your deployment will experience. As Iroha performs most of the operations in-memory, it is recommended to revise and increase the amount of available RAM or consider Optane-based solutions.
+
+### Minimum (small-scale deployment)
+
+Network connection, a 1 core CPU with 1GB of RAM. 1GB of persistent storage.
+
+### Recommended (enterprise-grade single server)
+
+High-bandwidth network connection, a 64 core single-socket or dual-socket CPU(s) with 512GB of ECC-capable RAM and 2TB of persistent storage
+
```

Comment:
Can be table style, such as Minimum/Recommended rows and CPU/RAM/storage/remarks columns